yis_gen.py

```diff
@@ -2170,6 +2194,12 @@ def computed_width(self):
             return self.width.computed_width
         return self.sv_type.computed_width
 
+    def computed_port_name(self, connection):
```

Comment:
I think this style would be more pythonic:

```suggestion
    def computed_port_name(self, connection):
      src = self.parent.block
      dst = connection['name']
      if self.direction == "input":
        src = connection['name']
        dst = self.parent.block
      return f"{src}__{dst}_{connection['submodule']}_{self.name}"
```

I think it's more typical python style to assign defaults, then check a conditional for an overwrite. I think it's more common in Perl (and probably other languages) to do a single line with multiple conditions.

Reply:
I'm fine with that, is there a way I can directly take your suggestion? or do I have to change code and re-push?

Reply:
I think there is a way to do it automatically, but you can copy the code out manually since there are only a few lines.

Reply:
done